lib/components/atoms/Tag/Tag.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+/**
+ *
+ * Tag
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+
+import Button from '../Button';
+import styles from './Tag.style';
+import type { Props } from './types';
+
+const Tag = ({ className, children, variation, onRemove }: Props): Node => {
+  const [showTag, setShowTag] = React.useState(true);
+
+  const removeTag = e => {
+    setShowTag(false);
+    if (onRemove) {
+      onRemove(e);
```

Comment:
Call back could also send which tag was removed. Else the remove won't mean much